modules/nf-core/cnvkit/batch/main.nf

```diff
@@ -27,7 +27,8 @@ process CNVKIT_BATCH {
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
+    def args = task.ext.args && task.ext.custom_args ? task.ext.args + ' ' + task.ext.custom_args :
```

Comment:
ooooh, so users wouldn't need to specify the whole args line anymore when overwritting it?

Reply:
yes, that's my idea, I wanted to do a POC so I can pitch to core and maintainers.

Reply:
Looking pretty good to me!